test/fs.test.js

```diff
@@ -161,4 +161,3 @@ describe('FileSystem based storage provider', function() {
     });
   });
 });
-
```

Comment:
don't your eslint rules forbid this kind of thing? Its not related to this PR, maybe you want to save this change for a PR to improve (or add?) the eslint config.